spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'Changing authentication factor' do
   describe 'requires re-authenticating' do
     before do
-      sign_up_and_2fa
+      @user = sign_up_and_2fa
```

Comment:
did we reach that decision? I heard 2 opinions (@jessie and @monfresh) on that.

Here's an opposite opinion.

1. `let()` is a basic feature of RSpec and is used extensively within the RSpec documentation and examples. https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/helper-methods/let-and-let
2. We have extensive use of `let()` already.
3. Given the existing usage in our project, in the RSpec standard, and in every other RSpec-based project I've seen, it seems to me to be a community idiom for creating temporary test fixtures in the way we have been doing it.
4. I don't understand the case for changing something that works, is well documented and widely used.